packages/editor/src/store/selectors.js

```diff
@@ -1748,3 +1748,20 @@ export function __experimentalGetTemplateInfo( state, template ) {
 		icon: templateIcon,
 	};
 }
+
+/**
+ * Returns a post type label depending on the current post.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {string|null} The post type label if available, otherwise null.
+ */
+export const getPostTypeLabel = createRegistrySelector(
+	( select ) => ( state ) => {
+		const currentPostType = getCurrentPostType( state );
+		const postType = select( coreStore ).getPostType( currentPostType );
+		// Disable reason: Post type labels object is shaped like this.
+		// eslint-disable-next-line camelcase
+		return postType?.labels?.singular_name || null;
```

Comment:
Is there a reason for explicitly returning `null` here? We can remove this, no?

Reply:
Yes, that's me just being a control freak and being explicit. 

Optional chaining will return `undefined` anyway.

Removed ✅